SpywareFilter/sections/general_url.txt

```diff
@@ -5,6 +5,8 @@
 ! Bad: .org/ads/$domain=example.org (for instance, should be in speficic.txt)
 !
 !
+! server-side GTM pattern B ex. https://sgtm.tagmanageritalia.it/kxacofiy.js?st=MK7842
+/\/[a-z]{8}\.js\?st=[0-9A-Z]{6,8}$/$script,~third-party,match-case
```

Comment:
what about to add checking for subdomain? They probably don't hosted on the main domain.
```
https://load.ss.esade.edu/ykpgwaiz.js?st=5V4W3ZR
https://sgtm.tagmanageritalia.it/kxacofiy.js?st=MK7842
```

Reply:
Sure, usually they're on a dedicated subdomain but this itself is not an unique characteristics of SSGTM. uBO has a filter option for this, [strict1p](https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#strict1p), which may be useful, though should not be relied on.

Reply:
https://regex101.com/r/PFhQyP/1 ?

Reply:
So `\/[a-z]{8}\.js\?st=[0-9A-Z]{6,8}$` is better

Reply:
Why? It does not check subdomain(s). Less wide regexp is better here.

Reply:
btw `www` may be exluded.

Reply:
> btw `www` may be exluded.

But using lookahead makes it incompatible with products on MV3 and Safari.

Reply:
`\/\/([a-z]{2,}\.){1,}[a-z]+\.[a-z]{2,}\/[a-z]{8}\.js\?st=[0-9A-Z]{6,8}$`
another possible

Reply:
Changed and also added a rule for regex where, IIRC, `{n,}` is not supported. The last `[0-9A-Z]{6,8}` seems to be most often `7`.